README.md

```diff
@@ -1,44 +1,24 @@
-# TFLint Ruleset Template
-[![Build Status](https://github.com/terraform-linters/tflint-ruleset-template/workflows/build/badge.svg?branch=main)](https://github.com/terraform-linters/tflint-ruleset-template/actions)
+# TFLint kafka-config ruleset
 
-This is a template repository for building a custom ruleset. You can create a plugin repository from "Use this template". See also [Writing Plugins](https://github.com/terraform-linters/tflint/blob/master/docs/developer-guide/plugins.md).
-
-## Requirements
-
-- TFLint v0.42+
-- Go v1.22
+This is a [tflint](https://github.com/terraform-linters/tflint) [plugin](https://github.com/terraform-linters/tflint/blob/master/docs/developer-guide/plugins.md) for enforcing UW rules over our kafka config.
 
 ## Installation
 
-TODO: This template repository does not contain release binaries, so this installation will not work. Please rewrite for your repository. See the "Building the plugin" section to get this template ruleset working.
-
 You can install the plugin with `tflint --init`. Declare a config in `.tflint.hcl` as follows:
 
 ```hcl
-plugin "template" {
+plugin "kafka-config" {
   enabled = true
 
-  version = "0.1.0"
-  source  = "github.com/terraform-linters/tflint-ruleset-template"
-
-  signing_key = <<-KEY
-  -----BEGIN PGP PUBLIC KEY BLOCK-----
-  mQINBGCqS2YBEADJ7gHktSV5NgUe08hD/uWWPwY07d5WZ1+F9I9SoiK/mtcNGz4P
-  JLrYAIUTMBvrxk3I+kuwhp7MCk7CD/tRVkPRIklONgtKsp8jCke7FB3PuFlP/ptL
-  SlbaXx53FCZSOzCJo9puZajVWydoGfnZi5apddd11Zw1FuJma3YElHZ1A1D2YvrF
-  ...
-  KEY
+  version = "x.y.z"
+  source  = "github.com/utilitywarehouse/tflint-ruleset-kafka-config"
 }
 ```
 
 ## Rules
 
 |Name|Description|Severity|Enabled|Link|
 | --- | --- | --- | --- | --- |
```

Comment:
I guess we're just leaving this a placeholder to be filled in later?

Reply:
yep